engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetCullFace.java

```diff
@@ -15,38 +15,23 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
-import org.lwjgl.opengl.GL11;
-import static org.lwjgl.opengl.GL11.GL_BACK;
-import static org.lwjgl.opengl.GL11.GL_FRONT;
-import static org.lwjgl.opengl.GL11.GL_FRONT_AND_BACK;
+import static org.lwjgl.opengl.GL11.GL_CULL_FACE;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
-import org.terasology.rendering.dag.tasks.SetCullFaceTask;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
+import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
 
 /**
  * TODO: Add javadocs
  */
-public class SetCullFace implements StateChange {
+public final class SetCullFace extends SetCapability {
```

Comment:
Think about the node developer. He needs to work on the desired state changes. He has opengl knowledge and knows he needs to enable face culling. What's going to be most intuitive for him to use: EnableFaceCulling() and SetWhatFacesToCull(mode) or SetCullFace(boolean) and SetCullFaceMode(int)?